python/cuml/neighbors/kneighbors_classifier.pyx

```diff
@@ -151,21 +152,14 @@ class KNeighborsClassifier(NearestNeighbors):
 
         # Only need to store index if fit() was called
         if self.n_indices == 1:
-            state['y'] = cudf.Series(self.y)
```

Comment:
Now that no special logic is required to serialize the classifier and regressor models, I don't believe they even need to have `__setstate__` and `__getstate__` functions at all.